### PR TITLE
Add Origin Trial mention for Web Bluetooth

### PR DESCRIPTION
**Pull request by beaufortfrancois**
@jyasskin Can you have a look? If that looks good to you, I'll ask @petele to merge and push live.

This is the same PR as #3142

### CODE REVIEW

**Comment by petele** [comment]
@beaufortfrancois - when this is ready, remove the DO NOT MERGE flag and I'll merge.

File: src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown [hunk 1]
```diff
@@ -44,7 +44,7 @@ finalized yet, the Chrome Team is actively looking for enthusiastic developers
 [feedback on the implementation](https://bugs.chromium.org/p/chromium/issues/entry?components=Blink%3EBluetooth).
 
 Web Bluetooth API is at the time of writing partially implemented in Chrome OS,
-Chrome for Android and Chrome for Linux behind an experimental flag. Go to
+Chrome for Android M, Linux and Mac behind an experimental flag. Go to
```
Comment (jyasskin) [inline src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown]: Oxford comma: "... Android M, Linux, and Mac"

File: src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown [hunk 2]
```diff
@@ -55,6 +55,19 @@ disconnected](#get-disconnected-from-a-bluetooth-device).
 
 <img style="width:723px; max-height:250px" src="/web/updates/images/2015-07-22-interact-with-ble-devices-on-the-web/web-bluetooth-flag.png" alt="Web Bluetooth Flag highlighted in chrome://flags"/>
 
+### Available for Origin Trials
+
+In order to keep from prematurely baking the Web Bluetooth API design in before
```
Comment (jyasskin) [inline src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown]: I might say this more positively: "In order to get as much feedback as possible from developers using the Web Bluetooth API in the field, we're also adding this feature in Chrome 53 as an origin trial for Chrome OS, Android M & Mac. During the origin trial, the API may still change in backward-incompatible ways before we freeze it into the web platform."

**Comment by jyasskin** [comment]
Otherwise, looks good.

File: src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown (hunk 1 above)
Comment (beaufortfrancois) [inline src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown]: I always forget this one. Thank You @jyasskin!
Fixed in https://github.com/google/WebFundamentals/pull/3147/commits/1fe13c7edc08cf9a94be9f25a6e88c2cf8764d98

**Comment by beaufortfrancois** [comment]
@jyasskin Thanks! I'll remove `DO NOT MERGE` later.

**Comment by beaufortfrancois** [comment]
@petele Flag is removed. Please merge and push live ;)

**Comment by petele** [comment]
will do a push tomorrow morning.

**Comment by owencm** [comment]
Few comments on contents:

It would be nice to state that the feature is available as an origin trial or behind a flag when first discussing those things. Today as I read it, the article makes it sound like the feature is only available behind a flag.

For example, we could swap: "Web Bluetooth API is at the time of writing partially implemented in Chrome OS, Chrome for Android M, Linux, and Mac behind an experimental flag. " with something like "Web Bluetooth API is currently available to be enabled experimentally on your origin an Origin Trial, or locally on your machine using an experimental flag. The implementation is partially complete and currently available on Chrome OS, Chrome for Android M, Linux, and Mac."

In the origin trials section it would be nice to add one sentence summarizing what origin trials are. So for example we could swap "In order to get as much feedback as possible from developers using the Web Bluetooth API in the field, we’re also adding this feature in Chrome 53 as an origin trial for Chrome OS, Android M, and Mac. During the origin trial, " with "In order to get as much feedback as possible from developers using the Web Bluetooth API in the field, we’re also adding this feature in Chrome 53 as an origin trial for Chrome OS, Android M, and Mac. Origin trials allow you to temporarily enable the feature for all of users of your website. During the origin trial, "

**Comment by beaufortfrancois** [comment]
@owencm I've addressed your feedback at https://github.com/google/WebFundamentals/pull/3184

**Pull request merged**